libraries/AP_HAL_Linux/RCInput_SBUS.cpp

```diff
@@ -14,9 +14,9 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 /*
-  this is a driver for DSM input in the QFLIGHT board. It could be
-  extended to other boards in future by providing an open/read/write
-  abstraction
+  this is a driver for SBUS input in Linux board using a UART. Note
+  that it relies on kernel support for 100kbaud and on a uart inverted
+  in hardware
```

Comment:
All of them are inverted UART? Otherwise you may need a spare gpio to
trigger an inverter in the middle while switching protocols.

Em 13 de jul de 2016 1:46 AM, "Andrew Tridgell" <notifications@github.com>
escreveu:
>
> In libraries/AP_HAL_Linux/RCInput_SBUS.cpp:
>
> > @@ -14,9 +14,9 @@
> >     along with this program.  If not, see <http://www.gnu.org/licenses/
>.
> >   */
> >  /*
> > -  this is a driver for DSM input in the QFLIGHT board. It could be
> > -  extended to other boards in future by providing an open/read/write
> > -  abstraction
> > +  this is a driver for SBUS input in Linux board using a UART. Note
> > +  that it relies on kernel support for 100kbaud and on a uart inverted
> > +  in hardware
>
> thanks. I still need to add DSM support for Disco, and auto-detection of
what RC input type is being used. I'd also like SUMD protocol support
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.